fix: 이벤트 카드 오류 해결

xaexunxang opened the pull request:
## 해시태그 캐러셀 도입
<img width="503" alt="image" src="https://github.com/user-attachments/assets/f06e055c-f67c-4cac-9cde-5c8a8d4218db" />
<img width="503" alt="image" src="https://github.com/user-attachments/assets/eba90b83-0fad-4372-b42f-f20a6e299068" />

- 호버할 때만 화살표 버튼이 보이도록 구현
- 가장 왼쪽에 있을 때는 오른쪽 화살표만 보임
- 가장 오른쪽에 있을 때는 왼쪽 화살표만 보임

## 삭제한 이벤트에 따른 티켓 상태값 동기화
<img width="503" alt="image" src="https://github.com/user-attachments/assets/889585c7-ab8e-4608-9645-ab5c4f658669" />

## 삭제 및 종료된 이벤트에 대한 티켓 블락처리
<img width="503" alt="image" src="https://github.com/user-attachments/assets/c57ec4e2-114b-488e-83dd-9e6c87ae7db4" />

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

* **New Features**
  * 해시태그를 가로로 스크롤할 수 있는 해시태그 캐러셀 컴포넌트가 추가되었습니다.
  * 이벤트 종료 또는 삭제 시 안내 모달이 표시됩니다.
  * 텍스트 모달 컴포넌트가 새롭게 추가되었습니다.

* **Enhancements**
  * 티켓, 이벤트, 카운트다운 등 주요 컴포넌트에 이벤트 상태(진행중, 종료, 삭제) 표시 및 처리 기능이 추가되었습니다.
  * 아이콘 버튼에 커스텀 CSS 클래스 적용이 가능해졌습니다.
  * 이벤트 슬라이더 내비게이션 버튼의 위치 지정이 개선되었습니다.
  * 이벤트 태그 섹션이 그리드 레이아웃으로 개선되었습니다.

* **Style**
  * Tailwind에 이벤트 종료 관련 색상 토큰이 추가되었습니다.

* **Dependencies**
  * classnames, react-virtuoso 패키지가 새로 추가되었습니다.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

xaexunxang committed:
chore: floating button 관련 패키지 설치

xaexunxang committed:
fix: 이벤트 해시태그 오류 캐러셀 구현으로 해결

xaexunxang committed:
refact: 이벤트 상태에 따른 티켓 상태 로직 추가

xaexunxang committed:
refact: 삭제되거나 종료된 이벤트에 따른 블락 및 모달창구현

xaexunxang committed:
fix: 전체 이벤트 보러가기 버튼 간격 조정

Yejiin21 reviewed (approved):
에러 해결하느라 고생하셨습니다!
코드래빗 리뷰 반영 후 리베이스 해주세요:)

xaexunxang committed:
fix: 이벤트 태그에 따른 섹션의 중복되는 CSS 제거

xaexunxang committed:
refact: 일관된 상태 관리를 위핸 setTimeout 함수 제거 및 React 상태 관리 패턴 사용

xaexunxang commented:
Finally!

xaexunxang committed:
# <타입> : <제목> 형식으로 작성하며 제목은 최대 50글자 정도로만 입력

# 제목을 아랫줄에 작성, 제목 끝에 마침표 금지, 무엇을 했는지 명확하게 작성
fix: API 호출 URL 제거 및 불필요한 콘솔 제거
################
# 본문(추가 설명)을 아랫줄에 작성

################
# 꼬릿말(footer)을 아랫줄에 작성 (관련된 이슈 번호 등 추가)

################
# feat: 새로운 기능 추가
# fix: 버그 수정
# docs: 문서 수정
# test: 테스트 코드, 리팩토링 테스트 코드 추가
# refact: 코드 리팩토링
# style: 코드 formatting, 세미콜론 누락, 코드 자체의 변경이 없는 경우
# chore: 패키지 매니저 수정, 그 외 기타 수정 ex) .gitignore
# design: CSS 등 사용자 UI 디자인 변경
# comment: 필요한 주석 추가 및 변경
# rename: 파일 또는 폴더 명을 수정하거나 옮기는 작업만인 경우
# remove: 파일을 삭제하는 작업만 수행한 경우
# !BREAKING CHANGE: 커다란 API 변경의 이유
# !HOTFIX: 급하게 치명적인 버그를 고쳐야 하는 경우

################
